UnitsNet/QuantityFormatter.cs

```diff
@@ -170,7 +170,7 @@ private static string FormatUntrimmed<TUnitType>(IQuantity<TUnitType> quantity,
                 {
                     var abbreviations = UnitsNetSetup.Default.UnitAbbreviations.GetUnitAbbreviations(quantity.Unit, formatProvider);
 
-                    if (abbreviationIndex >= abbreviations.Length)
+                    if (abbreviationIndex >= abbreviations.Count)
```

Comment:
This is the unfortunate side effect of moving away from the arrays, but there is no way around it..

I'm merging it..